web/src/components/AppFooter.vue

```diff
@@ -5,41 +5,90 @@
     app
     absolute
   >
-    <v-container>
+    <v-container
+      fluid
+    >
       <v-row no-gutters>
```

Comment:
Both `justify` and `align` properties refer to the content inside the element. `justify` sets the horizontal alignment, `align` sets the vertical alignment.

I tried this inspecting your branch on Chrome and it worked as you want:
- Add the `:justify="$vuetify.breakpoint.mdAndUp ? left : center"` logic here on this outer row
- Add `d-flex justify-center` to each `col` inside this row (cols need the `d-flex` property because it's not the default like it is for rows)
- Remove the extra row inside each col (they're not necessary) and leave the content directly inside the col

Reply:
Can you push a branch of what you tried? I'm getting a mess when I take out the inner rows.

<img width="1440" alt="SpeakHer" src="https://user-images.githubusercontent.com/4602369/95088995-97cc4900-075e-11eb-86b1-a23399a3f63f.png">

Reply:
Also I tried googling `d-flex` and that looks like Bootstrap notation... this is Vuetify

Reply:
Ahh I figured it out, I need to make 'left' and 'center' strings otherwise they resolve to undefined.